### PR TITLE
Add MagicaVoxel_mac.app v0.97

### PR DESCRIPTION
**Pull request by jodelamo**
[MagicaVoxel](https://ephtracy.github.io/)

### CODE REVIEW

File: Casks/magicavoxel.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 :no_check
+
+  url "http://23.98.147.40/uploads/MagicaVoxel-#{version}-win-mac.zip"
+  name 'MagicaVoxel'
+  homepage 'https://ephtracy.github.io/'
+  license :gpl
+
+  app 'MagicaVoxel_mac.app'
+
+  uninstall delete: '/Applications/MagicaVoxel_mac.app'
```
Comment (vitorgalvao) [inline Casks/magicavoxel.rb]: We do this automatically. Also, you can’t be sure users are installing here. Please remove it.

File: Casks/magicavoxel.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'magicavoxel' do
+  version '0.97'
+  sha256 :no_check
+
+  url "http://23.98.147.40/uploads/MagicaVoxel-#{version}-win-mac.zip"
```
Comment (vitorgalvao) [inline Casks/magicavoxel.rb]: Needs a `# 23.98.147.40 is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/magicavoxel.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cask 'magicavoxel' do
+  version '0.97'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/magicavoxel.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

**Comment by jlowgren** [comment]
@vitorgalvao Hopefully remedied the things you pointed out. Thanks for the helpful pointers!

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as ca947035f54f45d9a72c94bf2a20501f030911bc to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**